### PR TITLE
Desktop menu

### PR DESCRIPTION
**Pull request by Daphineatim**
In this milestone, I made changes to come up with a desktop version  of my Portfolio using media queries..

### CODE REVIEW

**Review by roshan-bajgain — COMMENTED** [review]
Hi @Daphineatim, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter fails. Kindly fix the linter and submit it again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -10,12 +7,23 @@
...
+    Ubuntu,
+    Cantarell,
+    "Open Sans",
+    "Helvetica Neue",
+    sans-serif;
   line-height: 1.5;
   font-size: 0.875rem;
   min-height: 100vh;
   max-width: 100vw;
   margin: 8px;
```
Comment (levy002) [inline style.css]: - [x] Kindly remove this margin to make your content look exactly like the provided Figma design. :100: 

  - ### Your design
   image


  - ### Figma design
   image

File: style.css [hunk 2]
```diff
@@ -108,26 +107,544 @@ a {
...
+    color: #344563;
+  }
+
+  .bg-image-1 {
+    background-image: url("./assets/desktop-header.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    border: 1px solid transparent;
+  }
```
Comment (levy002) [inline style.css]: - [x] Kindly make sure that all screen that has `min-width: 786px` (as it is your media query breakpoint) their headline section looks exactly as provided in the Figma :wink: I notice that everything starts to work fine at `min-width: 1055px`. Kindly make sure that at the breakpoint you had set `min-width: 786px` even there everything is working fine :wink: 

   - ### Your design (width: 787px)
   image


  - ### Figma design

File: style.css [hunk 3]
```diff
@@ -108,26 +107,544 @@ a {
...
+    padding: 0 20px;
+  }
+
+  /* works */
+  .works {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-row-gap: 120px;
+    padding: 120px 180px;
+  }
```
Comment (levy002) [inline style.css]: - [x] Kindly make to adapt your website to the desktop version as you had already set that breakpoint of `min-width: 786px` to be the desktop version. Please make sure to remove that padding to adapt your work container correctly. :100: 
- [x] Kindly make sure the project cards look exactly like the one on the Figma template.

     - ### Your design (width: 787px)
    image


  - ### Figma design
   image

File: style.css [hunk 4]
```diff
@@ -108,26 +107,544 @@ a {
...
+    flex-direction: row;
+    align-items: flex-start;
+  }
+
+  .div-container {
+    margin: 0 auto;
+    width: 100%;
+    padding: 10%;
+    gap: 5%;
+  }
```
Comment (levy002) [inline style.css:585]: - [x] Kindly make to adapt your website to the desktop version as you had already set that breakpoint of `min-width: 786px` to be the desktop version. Please make sure to adjust your margin and width to this `about-section` to look exactly as how it looks on the Figma design. :100: 

    - ### Your design (width: 787px)
     image


  - ### Figma design
   image

File: style.css [hunk 5]
```diff
@@ -108,26 +107,544 @@ a {
...
+}
+
+/* form */
+.form {
+  display: flex;
+  margin: 20px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
```
Comment (levy002) [inline style.css]: - [x] Kindly make sure that your `Get in touch` form button is aligned in the center like in the Figma design. :wink: 

     - ### Your design (width: 787px)
     image


  - ### Figma design
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @Daphineatim     ,  :wave: 
### I'm Levy, your PR reviewer

Great job so far :+1:  
However, they are some issues that you need to address before you move on to the next project :wink:

## Highlights
- Good job for using the correct Github flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
### Status: REQUESTED CHANGES :recycle: 

_Check my comments under review_

- [x] Kindly use the correct fonts to your text to look like the design provided. We can always find this special font family on [google fonts](https://fonts.google.com/?query=poppins). We have to import them from there and we can be able to show our text on the website with that font family.
   
     For your design, you have to use `font-family: 'Poppins', sans-serif` on your text but first of all, you need to import this font at the top of your CSS file. Please put this line there `@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,700&display=swap');` Now you will be able to use it as you want. :wink: 
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

 - [OPTIONAL] Kindly make sure to add a good PR description to the next projects. It makes your repo looks more professional. :wink: 
     - Example of good PR
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Daphineatim) [reply style.css]: Resolved!

File: style.css (hunk 2 above)
Comment (Daphineatim) [reply style.css]: Resolved

File: style.css (hunk 5 above)
Comment (Daphineatim) [reply style.css]: Resolved

File: style.css (hunk 4 above)
Comment (Daphineatim) [reply style.css:585]: Resolved. Thank you!

**Pull request merged**